packages/failurechecker/package.json

```diff
@@ -22,7 +22,7 @@
     "start": "probot run ./build/src/failurechecker.js",
     "start:local": "node ./build/src/local.js",
     "pretest": "npm run compile",
-    "test": "cross-env LOG_LEVEL=fatal c8 mocha build/test",
+    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit build/test",
```

Comment:
_grumble_ usually you only need that when a web-server connection isn't getting cleaned up.  I look at that as code smell.

Reply:
That wasn't meant to stay. While writing the tests, probot is retrying api requests that fail (unlocked) with backoff and I temporarily bumped the timeout to make them appear. I'll revert this part

Reply:
I have a fix for this coming

Reply:
This is now removed.